src/main/archetype/ui.content/src/main/content/jcr_root/conf/__appId__/settings/cloudconfigs/fdm/salesforce-cloud-service/.content.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
+    jcr:primaryType="cq:Page">
+    <jcr:content
+        cq:lastModified="{Date}2020-12-21T22:03:43.978+05:30"
+        cq:lastModifiedBy="admin"
+        jcr:lastModified="{Date}2020-12-21T22:03:43.915+05:30"
+        jcr:lastModifiedBy="admin"
+        jcr:primaryType="cq:PageContent"
+        jcr:title="Salesforce Cloud Service"
+        sling:resourceType="fd/fdm/gui/components/admin/fdmcloudservice/rest"
+        access_token_uri="https://login.salesforce.com/services/oauth2/token"
+        authenticationType="OAuth 2.0"
```

Comment:
@Meet001 Some one would need to manually do oAuth for this to work, right? So how are we proposing this as reference?

Reply:
@sakshi-arora1 Yes Customer would need to fill their MS Dynamics/Salesforce instance details and connect to oauth in properties page. this is similar to the way it was done for MS Dynamics in 6.5 https://experienceleague.adobe.com/docs/experience-manager-65/forms/form-data-model/ms-dynamics-odata-configuration.html?lang=en#configure-cloud-service-for-your-microsoft-dynamics-service